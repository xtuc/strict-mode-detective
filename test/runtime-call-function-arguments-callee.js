function f() {
  arguments.callee;
}

f();
