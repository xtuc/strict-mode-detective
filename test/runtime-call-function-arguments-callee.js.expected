  1 | function f() {
> 2 |   arguments.callee;
    |  ^^^^^^^^^^^^^^^^
  3 | }
  4 | 
  5 | f();
potentially illegal arguments.callee usage 

